README.md

```diff
@@ -150,4 +150,14 @@ Now that you have a basic sense of how to read these error messages, as well as
 
 When you're done, run `learn` to confirm all tests are passing.
 
+#### What's the Difference between `gets`, `gets.chomp`, and `gets.strip`?
+
+When getting user input, you may want to remove whitespace. The `gets.chomp` method removes whitespace AFTER the user input. The `gets.strip` method removes whitespace BEFORE and AFTER the user input. The `gets` method by itself does not remove any whitespace or newlines from the user input. Your tests will pass using any of these methods, however, be aware that they perform different actions and are not always interchangeable for every use-case.
```

Comment:
I'd also suggest cutting last sentence. If we know that this is a problem we should be able to fix a test.

Reply:
@sgharms researched on how to do this, haven't found a solution of how to force someone to use a specific method. This is an "issue" in a number of lessons. I don't know if rspec or other test suites are designed to test in this manner.